tests/HypothesisClient/Clock/ClockTest.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace tests\eLife\HypothesisClient\Clock;
+
+use eLife\HypothesisClient\Clock\Clock;
+
+/**
+ * @covers \eLife\HypothesisClient\Clock\Clock
+ */
+class ClockTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @test
+     */
+    public function it_can_return_the_current_time()
+    {
+        $clock = new Clock();
+        $this->assertEquals(time(), $clock->time());
```

Comment:
This test mirrors the implementation (http://jasonrudolph.com/blog/2008/07/30/testing-anti-patterns-the-ugly-mirror/)
Since the result is non-deterministic by design, I would relax the expectation to be `$this->assertGreaterThan(0, $clock->time());`